Code/GraphMol/FileParsers/ProximityBonds.cpp

```diff
@@ -74,9 +74,7 @@ bool IsBlacklistedPair(Atom *beg_atom, Atom *end_atom) {
 
     AtomPDBResidueInfo *beg_info = (AtomPDBResidueInfo *)beg_atom->getMonomerInfo();
     AtomPDBResidueInfo *end_info = (AtomPDBResidueInfo *)end_atom->getMonomerInfo();
-    if(!beg_info || !beg_info->getMonomerType() != AtomMonomerInfo::PDBRESIDUE)

```

Comment:
It seems to me like the test for the monomer type being PDBRESIDUE makes sense. Could it be that the problems encountered in the Travis build for this commit happened because it should be: `if(!beg_info || beg_info->getMonomerType() != AtomMonomerInfo::PDBRESIDUE)`?

Reply:
Oh, nice catch. For some reason after failure I assumed that HETATM got `AtomMonomerInfo::OTHER` type, so I ditched that check. It went in back - I did a rebase and merged the last comits.